index.rst

```diff
@@ -437,28 +486,29 @@ calls are made to DAX, the access token is passed as an OAuth 2.0 Bearer token i
 Notebook
 ~~~~~~~~
 
-The Portal and the notebook MAY share some common session information about the user, including
-refresh tokens, to enable smooth transitions and interoperability between the two. How this is
-implemented is undefined.
-
-Once a user is logged in to the Notebook access, a user in the Notebook aspect can be viewed as a
-special case of `data access libraries <#data-access-libraries>`__, where we have some access to the
-user's local environment, so we may be able to bootstrap an authentication mechanism on behalf of
-the user which ensures any necessary tokens are implicitly available in the user's environment. For
-software developed by the LSST that may utilize the LSP API aspect services, such as the Butler, we
-will ensure those applications can be automatically configured based on some form of information in
-the user's Notebook environment. Other third party software MAY be automatically configured, or they
-should be configurable in the same way as if a user was running on their local machine and not in an
+The portal and the notebook will share the same login flow, both being behind the `token proxy
+<#token-proxy>`__. Once the login has progressed past the token proxy and to the notebook, the
+notebook will initiate a notebook session based on the token that it has received. The notebook
+can then make the token available in the user's notebook environment.
+
+Once a user is logged in to the notebook aspect, with the token in the user's environment, a user
+in the notebook aspect can be viewed as a special case of `data access libraries
+<#data-access-libraries>`__, where we have some access to the user's local environment, so we may
+be able to bootstrap an authentication mechanism on behalf of the user which ensures any
+necessary tokens are implicitly available in the user's environment. For software developed by
+the LSST that may utilize the LSP API aspect services, such as the Butler, we will ensure those
```

Comment:
Remove "the" before LSST; change "may utilize" to "utilizes"; remove "such as the Butler,".